homeassistant/helpers/entity.py

```diff
@@ -657,7 +658,7 @@ def _substitute_name_placeholders(self, name: str) -> str:
             return name.format(**self.translation_placeholders)
         except KeyError as err:
             if not self._name_translation_placeholders_reported:
-                if get_release_channel() != "stable":
+                if get_release_channel() is ReleaseChannel.STABLE:
```

Comment:
This looks wrong to me.

Reply:
Please avoid giving subjective feedback without no indication of what is wrong with this. I missed the to properly translate from `!=` to `is not` but from the comment there was no clue at all on what was actually wrong leading to spending more time inferring the issue. Now it should be solved.